incubator/cloudflare-datadog/templates/deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "fullname" . }}
+spec:
+  replicas: {{default 1 .Values.replicaCount}}
```

Comment:
It's better to add defaults to `values.yaml` only. Don't specify them here if not necessary.

Use consistent formatting: Space after `{{` and before `}}`.